packages/node-cli/package.json

```diff
@@ -22,7 +22,7 @@
 		"suitecloud": "./src/suitecloud.js"
 	},
 	"dependencies": {
-		"@oracle/suitecloud-cli-localserver-command": "^1.0.2",
+		"@oracle/suitecloud-cli-localserver-command": "^1.0.10",
```

Comment:
Why is version upgrade needed?

Reply:
Finally I found out that it's not needed, I'll revert it